### ISSUE TITLE
[@thumbtack/thumbprint-react]: Disabled Label Font Color does not Comply with WCAG 2.0

### ISSUE THREAD

**Issue by TTblee** [thumbtack/thumbprint#570]
Disabled Labels don't comply with WCAG 2.0 in terms of contrast to background (assuming the background is #FAFAFA, the text #D3D4D5). That means the labels are hard to read.

This can be seen here: https://thumbprint.design/components/checkbox/react/#section-disabled-checkboxes

--------------
From https://webaim.org/resources/contrastchecker/
> WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. WCAG 2.1 requires a contrast ratio of at least 3:1 for graphics and user interface components (such as form input borders). WCAG Level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text.

![Image 2019-11-25 at 10 55 16 AM](https://user-images.githubusercontent.com/34253100/69570082-e708f100-0f73-11ea-9969-e05cf0a026b7.png)

**Comment by danoc** [comment]
Hey @TTblee — Thanks for the issue. My understanding is that the contrast requirements technically don't apply to disabled elements. [Here's an interesting thread on this](https://github.com/w3c/wcag21/issues/805). That said, I also disagree with the standard for the reasons mentioned in those GitHub comments.

While @jonpkerwin is exploring colors, I'd be curious to understand where and when we use disabled radios and checkboxes. Should we even be displaying the form elements if they are disabled?

**Comment by TTblee** [comment]
Here's an example of a design we are using on Pro Onboarding.

A user is enrolled in a category already, and we want to display that, but not allow them to deselect from the list. The page allows you to enroll your business into more categories.

<img width="900" alt="Image 2019-12-02 at 3 40 07 PM" src="https://user-images.githubusercontent.com/34253100/70004164-4047c600-151a-11ea-9eb9-0653ef835b29.png">

**Comment by danoc** [comment]
@tomgenoni — This could be another good fix-it week task to take on.

**Comment by tomgenoni** [comment]
I did some research and, as you'd expect from the w3c Github issue linked to above, there is no consensus about what color a disabled "thing" should be. I think ours could be a little darker but it's in the ballpark as other sites (FWIW, Airbnb's is in fact even lighter). Thumbprint will be adding guidelines on Forms in the coming months and we'll revisit this and a number of other form related items at that time. 

But one likely recommendation will be to only show disabled states when absolutely necessary. @TTblee in the example you posted above we could redesign this to avoid using the disabled checkbox. For example, we could separate the previously chosen item and use a check icon or use words to effect "You've picked Electrical and Wiring Repair..." Currently the disabled state connotes a somewhat negative "you picked this and can't undo it" rather than an affirmation of the choice.

I'm going to close this for now and add it to the items to consider during the Form guidelines project. If you'd like to discuss it further @TTblee let me know.
